csrank/core/fate_linear.py

```diff
@@ -53,8 +55,8 @@ def _construct_model_(self, n_objects):
 
     def step_decay(self, epoch):
```

Comment:
This method does not really belong here - it’s a general function for SGD and not something specific to FATE